modules/core/src/main/java/org/apache/ignite/IgniteServices.java

```diff
@@ -576,13 +580,15 @@ public void deployMultiple(String name, Service svc, int totalCnt, int maxPerNod
      * @param name Service name.
      * @param <T> Service type.
      * @return all deployed services with specified name.
+     * @see ServiceConfiguration#setStatisticsEnabled(boolean)
+     * @deprecated Use the proxies like {@link #serviceProxy(String, Class, boolean)}. References to local services
+     * corrupt the service statistics and bring no real performance optimization.
      */
+    @Deprecated
     public <T> Collection<T> services(String name);
 
     /**
-     * Gets a remote handle on the service. If service is available locally,
-     * then local instance is returned, otherwise, a remote proxy is dynamically
-     * created and provided for the specified service.
+     * Gets a handle on the service.
```

Comment:
Let's keep a mention about remote services and proxy creation. 

For example, "Gets a handle on the remote or local service. Proxy is dynamically created and provided for the specified service".

Reply:
Fixed